forch/simple_auth_state_machine.py

```diff
@@ -1,39 +1,95 @@
 """Module that implements MAB state machine"""""
 
 import logging
+from threading import Lock
+import time
 
 LOGGER = logging.getLogger('mabsm')
 
 
 class AuthStateMachine():
     """Class represents the state machine that handles the Auth for a session"""""
 
+    UNAUTH = "Unauthorized"
+    REQUEST = "RADIUS Request"
+    ACCEPT = "Authorized"
+    MAX_RADIUS_RETRIES = 5
+    RADIUS_QUERY_TIMEOUT = 10
+    REJECT_STATE_TIMEOUT = 20
```

Comment:
very inconsistant names -- one is _QUERY_ one is _STATE_ and one is _SESSION_... try to provide a coherent naming structure.

Reply:
Addressed.